articles/azure-arc/servers/prerequisites.md

```diff
@@ -36,30 +36,59 @@ If two agents use the same configuration, you'll encounter inconsistent behavior
+| SUSE Linux Enterprise Server (SLES) | 15          | ✅    | ✅    | see [ARM64 Architecture support](#arm64-architecture-support)                                    |
+| Ubuntu                              | 18.04       | ✅    | ❌    |                                                                                                  |
+| Ubuntu                              | 20.04       | ✅    | ❌    |                                                                                                  |
+| Ubuntu                              | 22.04       | ✅    | ❌    |                                                                                                  |
+| Windows Client                      | 10          | ✅    | ❌    | see [client operating system guidance](#client-operating-system-guidance)                        |
+| Windows Client                      | 11          | ✅    | ❌    | see [client operating system guidance](#client-operating-system-guidance)                        |
+| Windows IoT Enterprise              |             |       |        |                                                                                                  |
+| Windows Server                      | 2008 R2 SP1 | ⚠️    | ❌    | **Limited Support**, see [Limited support operating systems](#limited-support-operating-systems) |
+| Windows Server                      | 2012        | ✅    | ❌    |                                                                                                  |
+| Windows Server                      | 2012 R2     | ✅    | ❌    |                                                                                                  |
+| Windows Server                      | 2016        | ✅    | ❌    |                                                                                                  |
+| Windows Server                      | 2019        | ✅    | ❌    |                                                                                                  |
+| Windows Server                      | 2022        | ✅    | ❌    |                                                                                                  |
+
+
+For Windows Server, both Desktop and Server Core experiences are supported. Azure Editions are supported on Azure Stack HCI.
 
 The Azure Connected Machine agent isn't tested on operating systems hardened by the Center for Information Security (CIS) Benchmark.
 
 > [!NOTE]
 > [Azure Connected Machine agent version 1.44](agent-release-notes.md#version-144---july-2024) is the last version to officially support Debian 10, Ubuntu 16.04, and Azure Linux (CBL-Mariner) 1.0.
 > 
 
+## ARM64 Architecture support
+
+Not all features, virtual machine extensions, and services are supported on ARM64 at this point. For full details on ARM64 compatibility, please
+refer to the documentation for the service you wish to use.
```

Comment:
Sure, I can include a few things we know work. I did specify that machine config doesn't work, as I felt that was important to mention. IMO it's best to avoid listing things like extensions that don't work, as that can change at any time which would make this outdated.